src/core.jl

```diff
@@ -154,10 +132,10 @@ function trainGigaSOM(som::Som, train;
                       rStart = 0.0, rFinal=0.1, radiusFun=linearRadius,
                       epochs = 10)
 
-    train = convertTrainingData(train)
+    train = Matrix{Float64}(train)
 
-    #this slices the data using `distribute` and sends them to workers
-    dInfo = distribute_darray(:trainDataVar, distribute(train))
+    #this slices the data into parts and and sends them to workers
+    dInfo = distribute_array(:GigaSOMtrainDataVar, train, workers())
     res = trainGigaSOM(som, dInfo,
                        kernelFun=kernelFun,
                        metric=metric,
```

Comment:
in line 146: can you rename the return object to `som` or `som_res`?

Reply:
good point. :]